extensions/objects/templates/sanitized-url-editor.component.spec.ts

```diff
@@ -84,4 +99,22 @@ describe('SanitizedUrlEditorComponent', () => {
     expect(component.valueChanged.emit).not.toHaveBeenCalledWith('http://oppia.org/');
     expect(detectChangesSpy).not.toHaveBeenCalled();
   });
+
+  it('should set form validity', fakeAsync(() => {
```

Comment:
This could use more explanatory scaffolding.

Reply:
Done